openshift/ci-operator/update-ci.sh

```diff
@@ -2,6 +2,9 @@
 # A script that will update the mapping file in github.com/openshift/release
 
 set -e
+
+source "$(dirname "$0")/../common.sh"
```

Comment:
`common.sh` is that a good name for the fancy ui? :smile:

Reply:
well it's just starting a common script that can include other functions, a convention been followed all over. I hardly see any "specific" sourced shell scripts . Check e2e-common etc. 

I've no strong opinion there, just saying it's same as we've been doing so far.